packages/adapter-netlify/README.md

```diff
@@ -66,38 +68,6 @@ During compilation a required "catch all" redirect rule is automatically appende
   node_bundler = "esbuild"
 ```
 
-## Advanced Configuration
```

Comment:
I wonder if we'll catch anyone out by removing this since we haven't actually entirely gotten rid of esbuild yet in the zip-it-and-ship-it case. It looks like Vercel always uses esbuild still and so we're probably even more likely to have people there who can no longer deploy with these options removed there

Reply:
from discord, discussing the original reason for exposing `esbuild` configuration: 

> so of those issues, the node-related ones should vanish since esbuild is no longer being used there. the sharp thing is a real problem for environments where we need to bundle (vercel, cloudflare) but giving people the ability to modify the esbuild config isn't going to change that AFAICT. the type reflection issue seems unrelated to any final bundling step, it was happening during dev i think? so it feels like we can probably remove the ability to modify the config, at least for now

there's just no reason to expose it that i can see. we could reinstate it if someone comes up with a valid use case, but right now there doesn't appear to be one